util/constants.js

```diff
@@ -0,0 +1 @@
+export const API_URL = "http://127.0.0.1:8000/"
```

Comment:
Why not use cops-hackalog.herokuapp.com even while development?

Reply:
Ohh yeah, right!

Reply:
Done